docs/services/vision/detection.md

```diff
@@ -22,14 +22,12 @@ The returned detections consist of the bounding box around the identified object
 - `confidence` (float): specifies the confidence of the assigned label.
   Between `0.0` and `1.0`, inclusive.
 
-## Detector Types
-
 You can use the following types of detectors:
 
-- [**color_detector**](#color_detector): A heuristic detector that draws boxes around objects according to their hue (does not detect black, gray, and white).
-- [**tflite_detector**](#tflite_detector): A machine learning detector that draws bounding boxes according to the specified .tflite model file available on the robot’s hard drive.
+- [**color_detector**](#configure-a-color_detector): A heuristic detector that draws boxes around objects according to their hue (does not detect black, gray, and white).
+- [**mlmodel**](#configure-a-mlmodel-detector): A machine learning detector that draws bounding boxes according to the specified .tflite model file available on the robot’s hard drive.
```

Comment:
Rather than being specifically about a tflite model, `mlmodel` should be able to use any ML Model service, whether it's a tflite model, or another model that the ML Model Service will one day support.